HISTORY.rst

```diff
@@ -1,3 +1,11 @@
+0.17.0 (XX-XX-XXXX)
```

Comment:
You need to modify CHANGES.rst instead. 0.17.0 is an upcoming release, not the archived one.